### ISSUE TITLE
 About website pictures comparison.

### ISSUE THREAD

**Issue by Likangkang08** [SeleniumHQ/selenium-ide#1029]
## 🚀 Feature Proposal

Compare whether the pictures on the website are consistent with the local ones。

## Motivation

The same version of the project may have different pictures, so you need to verify the picture of the project, hoping to provide a method of comparing pictures

## Example

For example, compare GitHublogo。
![image](https://user-images.githubusercontent.com/22090651/80575019-72ac0f80-8a35-11ea-8614-45652df59eca.png)

**Comment by corevo** [comment]
This will have to wait for the electron app since we will need fs access to get the expected images' blob.

**Comment by Likangkang08** [comment]
We can put the expected pictures in the designated directory in advance and compare the screenshots when running the program.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Tomer Steinfeld"<notifications@github.com&gt;; 
发送时间: 2020年4月29日(星期三) 晚上6:25
收件人: "SeleniumHQ/selenium-ide"<selenium-ide@noreply.github.com&gt;; 
抄送: "李康康"<1712340552@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [SeleniumHQ/selenium-ide]  About website pictures comparison. (#1029)





 
This will have to wait for the electron app since we will need fs access to get the expected images' blob.

**Comment by corevo** [comment]
Yes I know, but Selenium IDE can't access your filesystem to read the expected images.

**Comment by Likangkang08** [comment]
I think we can try to use python's opencv to compare the pictures. Selenium is only responsible for screenshots.

![image](https://user-images.githubusercontent.com/22090651/80678597-cda13d80-8aed-11ea-943d-9c41e1c151b7.png)
For example, before you start, put the original picture in the base file, take a screenshot at runtime, and then use python to write a method for comparison. If the same returns true, it returns false. Then mark the different places.
![image](https://user-images.githubusercontent.com/22090651/80678858-53bd8400-8aee-11ea-8308-fe1db963bb54.png)

**Comment by corevo** [comment]
You don't understand what I'm trying to tell you
- Selenium IDE can't run python (but we don't need to we can compare in js)
- Selenium IDE can't access your files

Because Selenium IDE is a browser extensions we have 2 fairly large limitations:
- We can't start processes
- We can't read & write files to your computer

Therefore this issue is stalled until we move to electron, if you find a way to run opencv from Selenium IDE, you are more than welcome to send a PR.

**Comment by Likangkang08** [comment]
I understand what you mean, thank you for your reply.
